utils/calibration.py

```diff
@@ -277,210 +280,118 @@ def calibration_IsotonicReg(tpred_samples_cal, data_cal, target_cal, sigmas_samp
 	return 1-conf_levels, unc_pcts, cal_pcts, unc_pcts2, cal_pcts2, isotonic
 
 
-def calibration_pdf22(tpred_samples, data_test, target_test, target_test2, sigmas_samples, position, alpha = 0.85, id_batch=-2, draw=False, gaussian=False):
+def gt_evaluation(target_test, target_test2, k, position, fk, s_xk_yk, gaussian=False, fk_max=1.0):
```

Comment:
Not super fan of this name

Reply:
The `calibration_pdf22` function was renamed as `calibration_density()`

The `calibration_pdf23` function was renamed as `calibration_relative_density`

The function `gt_evaluation()` is a new function that simple does that, returns the evaluated GT. I think since this new function was added at the same line as `calibration_pdf22` that's why it seems to be renamed, but it does not. Feel free to review the PR by commits so it's easier to identify this type of changes :)